### DISCUSSION TITLE
Alerting/Reporting Functionality

### DISCUSSION THREAD

**Discussion by ow111** [Security-Onion-Solutions/securityonion#2534] [Unsupported Versions] [answered]
Looking for ways of delivering Daily/Weekly reports for auditing through medium of Email/PDF Generation. 

Is this functionality already implemented and supported by SO? We have been trying to get ElastAlert working however having issues adding rules manually and testing therein. 

We have thought of adding extra software such as Skedler/Sentinl to support this functionality through kibana however reluctant to add stuff into the prebuilt SO package. 

Sorry If something has been missed here, Reporting proving to be our only stumbling block with SO

**Comment by weslambert** [comment, marked as answer]
Please see: https://github.com/Security-Onion-Solutions/securityonion/discussions/1902#discussioncomment-128679

**Reply by ow111** [reply]
Thanks very much

**Reply by achak17** [reply]
Skedler is an easy and affordable tool to automate Security Onion reports using Kibana. Check out this blog to know how: https://www.skedler.com/blog/automate-security-onion-reports-using-skedler/
